### DISCUSSION TITLE
Launching Neovim in a specific terminal program

### DISCUSSION THREAD

**Discussion by PremiereStoss** [neovim/neovim#27354] [Q&A] [answered]
I know this is going to be an easy question to answer.  

I just installed Arch Linux and I have setup Neovim.  When I click on the Neovim icon, it opens in 'Console'.  I want it to open in 'Terminator'.  How do I change that?

Thanks.

**Comment by zeertzjq** [comment, marked as answer]
1. Copy `/usr/share/applications/nvim.desktop` to `~/.local/share/applications/nvim.desktop`
2. Edit `~/.local/share/applications/nvim.desktop`, remove `Terminal=true` and change `Exec=nvim %F` to `Exec=terminator -e 'nvim %F'`

**Reply by PremiereStoss** [reply]
Thanks for the reply!

Unfortunately, I do not have an nvim.desktop.  I have a vim.desktop.  I searched all files and nowhere is it to be found.

**Reply by zeertzjq** [reply]
Maybe it's `/usr/local/share/applications/nvim.desktop` then.

**Reply by PremiereStoss** [reply]
Ok, I found it there.  I made the changes but now, Neovim icon does launch anything.

![image](https://github.com/neovim/neovim/assets/53406832/b2eac8a1-9796-4ee7-aed7-d9c80ca546f5)

**Reply by PremiereStoss** [reply]
sorry, didn't see that last little bit....trying now

**Reply by PremiereStoss** [reply]
Works perfectly, thanks so much!  Sorry, I didn't see that last little bit of code in your example.

Could you please tell me what the -e is for?

**Reply by zeertzjq** [reply]
Run `terminator -h` and you will see

**Reply by PremiereStoss** [reply]
Got it, makes sense now.  Thanks for all your help!
